components/OrchestratorList/index.tsx

```diff
@@ -111,6 +114,13 @@ const OrchestratorList = ({
     () => numeral(Number(principle) || 150).format("0a"),
     [principle]
   );
+  const { data: bondingManagerAddress } = useBondingManagerAddress(); 
```

Comment:
I've currently utilized the `bondingManager` contract to retrieve this information, but another option is to include it in the subgraph and use that data instead.